packages/app/app/components/HelpModal/index.js

```diff
@@ -48,6 +62,19 @@ const HelpModal = () => {
       <Modal.Content>
         <Modal.Description>{agplDisclaimer}</Modal.Description>
       </Modal.Content>
+      <Modal.Content>
+        <div className={styles.social_icons}>
+          <div className={styles.icon}>
+            <Icon link size={'big'} name='twitter' onClick={handleTwitterClick}/>
+          </div>
+          <div className={styles.mastadon}>
+            <Image src={mastadonImg} onClick={handleMastadonClick} />
+          </div>
```

Comment:
I really had trouble doing this with the svg. So I converted it to a png and went with that. If you have a better solution then I'd be glad to do it like that